### PR TITLE
Readme housekeeping

### PR DESCRIPTION
**Pull request by van-fs**
- Removed the "beta" notice
- Added the script location
- Made a few clarity adjustments for past fixes and changes

### CODE REVIEW

**Comment by van-fs** [comment]
Moved to draft while we validate the changes with PM.

**Comment by van-fs** [comment]
Green light from PM, sending to @TrevorFSmith

File: README.md [hunk 1]
```diff
@@ -255,16 +261,27 @@ After new rules are deployed, visit the site and ensure the data is being sent t
 
 ## Monitoring
 
-The Quick Start snippet provides a FullStory log appender.  When a log event is generated by DLO, it will be sent to FullStory as a custom event.  You should periodically review the `Data Layer Observer` custom event to monitor data layer events.
+The Quick Start snippet provides a FullStory log appender.  When a log event is generated by DLO, it will be sent to FullStory as a custom event.  You should periodically review the `Data Layer Observer` custom event to monitor data layer events.  Logging can be disabled by setting `window['_dlo_logLevel']` to the value `-1`.
```
Comment (patrick-fs) [inline README.md:264]: What's the default value?

Comment (van-fs) [reply README.md:264]: It defaults to `1`, which is the WARN level.  This is included in the config section as

```
_dlo_logLevel | number | 1 | Log messages at this level and below will be logged by the LogAppender. Defaults to WARN.
```

But let me make a clarifying statement of all values that can be set.

Comment (van-fs) [reply README.md:264]: Actually, looks like I had that already `level corresponds to the log level: -1 for none, 0 for error, 1 for warn, 2 for info, and 3 for debug.`  I may itemize it for readability, and I'm updating the screenshot.

**Review by TrevorFSmith — APPROVED** [review]

**Pull request merged**